setup.py

```diff
@@ -23,7 +23,7 @@
     "Programming Language :: Python :: Implementation :: PyPy",
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
-INSTALL_REQUIRES = ['devo-sdk>=3.0.3,<4', 'faker', 'jinja2>=2.10.1', 'psutil',
+INSTALL_REQUIRES = ['devo-sdk>=3.2.0,<4', 'faker', 'jinja2>=2.10.1', 'psutil',
```

Comment:
Please update here to 3.2.1 and fix version of all requires like i say in main comments. Thanks.

Reply:
Same as above